### ISSUE TITLE
Horrible Bengali (বাংলা) Language Translation

### ISSUE THREAD

**Issue by bauripalash** [LibreHealthIO/lh-ehr#1326]
## Problem

While checking the bleeding age demo, as soon as I shifted to Bengali Language, I was horrified the translation was horrible , most of them were wrong.

**Example:**
**Message and Reminder Center** Tab is translated to **মেসেজ** which is just **Message** written in Bengali Alphabets.

## Possible Fix :
If a little guidance provided I can improve the Bengali Language Translations.

**Comment by Trodrige** [comment]
@bauripalash start looking into it and ask us here to get all the help you need.

**Comment by KoniKodes** [comment]
@bauripalash Is this in the Hindi language? How do you suggest to help?

**Comment by bauripalash** [comment]
No, this is Bengali, but not correctly translated. I can improve the bengali translation

**Comment by KoniKodes** [comment]
Oh, I see. A previous applicant was working with Hindi, excuse my confusion.
How do you plan to improve the translation, and who else could follow-up to assure your translation is correct?

**Comment by bauripalash** [comment]
In of section II of `interface/language/lang.info.html` , it's mentioned the there is a Google Doc to maintain language data, if you gave enough privilege , I can improve the Bengali internationalization data over there.

And I've Google Translate and Some E2B online dictionaries to prove my accuracy. If still if there any Bengali Contributors I'd be happy to work with them

**Comment by aethelwulffe** [comment]
@bauripalash Go for it brother.  KoniKodes is the translations work central contact right now.

**Comment by aethelwulffe** [comment]
We can produce you a file to work on this.  We have some precursor stuff (like fixing the English in the actual code) to commit yet.  Spanish update is close, and we really need help with the Bengali...it is a total junk stub.

**Comment by KoniKodes** [comment]
Thank you @aethelwulffe and @bauripalash 
We'll be happy to be here and help when you are ready.
